odoo80/Dockerfile

```diff
@@ -25,12 +23,10 @@ RUN apt-get update -q && apt-get upgrade -q \
     postgresql-common \
     libpq-dev \
     libldap2-dev \
-    libsasl2-dev \
     libxml2-dev \
     libxslt1-dev  \
     xsltproc \
     xmlstarlet \
-    openssl \
     python-libxml2 \
     phantomjs \
     bash-completion
```

Comment:
@ruiztulio 
Yes, I can.
I started my comment with that: https://github.com/Vauxoo/docker-odoo-image/pull/70#issuecomment-182182516